framework/hooks/useClipboard.tsx

```diff
@@ -16,14 +17,15 @@ type UseClipboardResult = {
  * - copySuccess: A boolean indicating if the last copy attempt was
  * successful.
  */
-export function useClipboard(): UseClipboardResult {
+export function useClipboard(timeout: number | undefined = undefined): UseClipboardResult {
```

Comment:
```suggestion
export function useClipboard(timeout: number  = 2000): UseClipboardResult {
```

@lgalis What about we have a default value?

Reply:
@nixocio - changed for all the copy alerts to time out after the default value ( 8000 )